Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -630,6 +630,22 @@ public extension Purchases {
         return try await logOutAsync()
     }
 
+    ///
+    /// Updates the current appUserID to a new one, without associating the two.
+    /// - Important: This method is **only available** in Custom Entitlements Computation mode.
+    /// Receipts posted by the SDK to the RevenueCat backend after calling this method will be sent
+    /// with the newAppUserID.
+    ///
+    @objc(switchUserTo:)
+    func switchUser(to newAppUserID: String) {
```

Comment:
It definitely feels some users might inadvertently use this method and it will error silently for them. I wonder if we should just crash if they are not in custom entitlement computation mode... Also, moving this to a different package could avoid this confusion.

Reply:
Solved by #2440.